packages/flutter_test/test/widget_tester_test.dart

```diff
@@ -13,10 +13,14 @@ import 'package:flutter/rendering.dart';
 import 'package:flutter/scheduler.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter_test/src/leak_tracking.dart';
+import 'package:leak_tracker/leak_tracker.dart';
```

Comment:
This import surprises me and probably means we are missing some exports. If we build this into the testing framework, you should be able to use it without adding another dependency.